filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsPathBuilderSpec.scala

```diff
@@ -206,29 +178,6 @@ class DemoDosPathBuilderSpec extends TestKitSuite with FlatSpecLike with Matcher
       getNameCount = 0,
       isAbsolute = true),
 
-    GoodPath(
-      description = "a bucket with a normalized path ..",
-      path = s"dos://$bucket/..",
-      normalize = true,
```

Comment:
This test was removed, but I don't see it being added to the `BadPath` section below. If the path still resolves "successfully" in some `Good` way, then this test should just be updated, but not removed. Otherwise the expected error should be added to the `BadPath` section below.

Reply:
Yes, it does resolve 'successfully' after minor edits. So I am reverting the removal of test.